src/Microsoft.AspNet.Mvc.Core/Formatters/OutputFormatter.cs

```diff
@@ -251,21 +250,20 @@ private Encoding MatchAcceptCharacterEncoding(IList<StringWithQualityHeaderValue
         {
             if (acceptCharsetHeaders != null && acceptCharsetHeaders.Count > 0)
             {
-                var sortedAcceptCharsetHeaders = acceptCharsetHeaders
-                    .Where(acceptCharset => acceptCharset.Quality != HeaderQuality.NoMatch)
-                    .OrderByDescending(m => m, StringWithQualityHeaderValueComparer.QualityComparer);
-
-                foreach (var acceptCharset in sortedAcceptCharsetHeaders)
+                var sorted = Sort(acceptCharsetHeaders);
```

Comment:
Suggest `acceptValues` is better here as well. Liked that name in the next file.